lib/framereader.py

```diff
@@ -177,6 +177,17 @@ def read_frame_BGRA(fd, width, height):
 
   return r, g, b
 
+def read_frame_BGRX(fd, width, height):
+  size = width * height * 4
+
+  argb = numpy.fromfile(fd, dtype=numpy.uint8, count=size)
```

Comment:
`argb` variable name is not appropriate for this.  Use `bgrx`.

Reply:
updated